UI/src/view/LinearAssetLabel.js

```diff
@@ -27,4 +27,77 @@
         };
 
     };
+
+    root.LinearAssetLabelMultiValues = function(){
+
+        AssetLabel.call(this);
+
+        var me = this;
+        var IMAGE_HEIGHT = 27;
+
+        this.getStyle = function(value){
+          return createMultiStyles(value);
+        };
+
+        var createMultiStyles = function(values){
+          var i = 0;
+          var splitValues = values.replace(/\s/g,'').split(/[\n,]+/);
+          var styles = [];
+          _.forEach(splitValues, function(value){
+            i++;
+            styles.push(backgroundStyle(value, i), textStyle(value, i));
+          });
+          return styles;
+        };
+
+        var backgroundStyle = function(value, i){
+          var image = 'images/linearLabel_background.png';
+          if(!correctValues(value))
+            image = 'images/warningLabel.png';
+
+          return new ol.style.Style({
+            image: new ol.style.Icon(({
+              anchor: [17, (i * IMAGE_HEIGHT) - 15],
```

Comment:
maybe the value 17 and 15 can be declared as a variable like `var IMAGE_HEIGHT = 27;`